File: tirefitting/modules/core/src/com/company/tirefitting/core/ChangeWheel.java
package com.company.tirefitting.core;

import org.springframework.stereotype.Component;

@Component(ChangeWheel.NAME)
public class ChangeWheel implements RequestTypeDurationCalculator {
    public static final String NAME = "tirefitting_ChangeWheel";

    @Override
    public double getDuration(int radius) {
        return 15 * 60 * (1 + ((double)radius - 13) / 5);
    }
}

File: tire_fitting/lib/requestTypes/ChangeWheel.dart
import 'package:tire_fitting/requestTypes/RequestType.dart';

class ChangeWheel extends RequestType{
  @override
  double getDuration(int radius) {
    return 15 * 60 * (1 + (radius - 13) / 5);
  }

  String name = 'CHANGE_WHEEL';

}